fwprovider/acme/resource_acme_plugin.go

```diff
@@ -0,0 +1,326 @@
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64default"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/bpg/terraform-provider-proxmox/proxmox"
+	"github.com/bpg/terraform-provider-proxmox/proxmox/cluster/acme/plugins"
+)
+
+var (
+	_ resource.Resource                = &acmePluginResource{}
+	_ resource.ResourceWithConfigure   = &acmePluginResource{}
+	_ resource.ResourceWithImportState = &acmePluginResource{}
+)
+
+// NewACMEPluginResource creates a new resource for managing ACME plugins.
+func NewACMEPluginResource() resource.Resource {
+	return &acmePluginResource{}
+}
+
+// acmePluginResource contains the resource's internal data.
+type acmePluginResource struct {
+	// The ACME account API client
+	client plugins.Client
+}
+
+// acmePluginCreateModel maps the schema data for an ACME plugin.
+type acmePluginCreateModel struct {
+	// API plugin name
+	API types.String `tfsdk:"api"`
+	// DNS plugin data
+	Data types.Map `tfsdk:"data"`
+	// A list of settings you want to delete.
+	Delete types.String `tfsdk:"delete"`
+	// Prevent changes if current configuration file has a different digest.
+	// This can be used to prevent concurrent modifications.
+	Digest types.String `tfsdk:"digest"`
+	// Flag to disable the config
+	Disable types.Bool `tfsdk:"disable"`
+	// Plugin ID name
+	Plugin types.String `tfsdk:"plugin"`
+	// List of cluster node names
+	Nodes types.String `tfsdk:"nodes"`
+	// ACME challenge type (dns, standalone)
+	Type types.String `tfsdk:"type"`
+	// Extra delay in seconds to wait before requesting validation (0 - 172800)
+	ValidationDelay types.Int64 `tfsdk:"validation_delay"`
+}
+
+// Metadata defines the name of the resource.
+func (r *acmePluginResource) Metadata(
+	_ context.Context,
+	req resource.MetadataRequest,
+	resp *resource.MetadataResponse,
+) {
+	resp.TypeName = req.ProviderTypeName + "_acme_plugin"
+}
+
+// Schema defines the schema for the resource.
+func (r *acmePluginResource) Schema(
+	_ context.Context,
+	_ resource.SchemaRequest,
+	resp *resource.SchemaResponse,
+) {
+	resp.Schema = schema.Schema{
+		Description: "Manages an ACME plugin in a Proxmox VE cluster.",
+		Attributes: map[string]schema.Attribute{
+			"api": schema.StringAttribute{
+				Description: "API plugin name.",
+				Optional:    true,
+			},
+			"data": schema.MapAttribute{
+				Description: "DNS plugin data.",
+				Optional:    true,
+				ElementType: types.StringType,
+			},
+			"delete": schema.StringAttribute{
+				Description: "A list of settings you want to delete.",
+				Optional:    true,
+			},
+			"digest": schema.StringAttribute{
+				Description: "Prevent changes if current configuration file has a different digest. " +
+					"This can be used to prevent concurrent modifications.",
```

Comment:
```suggestion
				Description: "SHA1 digest of the current configuration.",
				MarkdownDescription: "SHA1 digest of the current configuration. Prevent changes if current configuration file has a different digest. " +
					"This can be used to prevent concurrent modifications.",
```